vip_spec.xsd

```diff
@@ -260,13 +265,14 @@ reference elements for polling location
           <xs:complexType>
             <xs:sequence>
               <xs:element name="ElectoralDistrictID" type="xs:IDREF" />
-              <xs:element name="ContactID" type="xs:IDREF" minOccurs="0" maxOccurs="unbounded" />
               <xs:element name="Name" type="InternationalizedText" />
               <xs:element name="ExternalIdentifierID" type="xs:IDREF" minOccurs="0" maxOccurs="unbounded" />
-              <xs:element name="Partisan" type="xs:boolean" minOccurs="0" />
-              <xs:element name="TermEndDate" type="xs:date" minOccurs="0" />
-              <xs:element name="TermStartDate" type="xs:date" minOccurs="0" />
-              <xs:element name="TermType" type="TermType" />
+              <xs:element name="OfficeHolder" type="xs:IDREF" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
Again, our style is to end in `[TypeName]ID` for ID references.  (By the way, I do think the guideline helps because it says what type the ID is a reference to.)